src/loaddb/load_server_loader.cpp

```diff
@@ -347,7 +347,12 @@ namespace cubload
 	  }
 
 	db_value &db_val = get_attribute_db_value (attr_index);
-	heap_attrinfo_set (&m_class_entry->get_class_oid (), attr.get_repr ().id, &db_val, &m_attrinfo);
+	error_code = heap_attrinfo_set (&m_class_entry->get_class_oid (), attr.get_repr ().id, &db_val, &m_attrinfo);
+	if (error_code != NO_ERROR)
+	  {
+	    m_error_handler.on_syntax_failure ();
```

Comment:
please check that old loaddb reported this syntax error

Reply:
old loaddb doesn't report any error since it uses different routine for domain compatibility, I'll leave it for now as syntax error.